controlplane/internal/controller/clusterdeployment_controller.go

```diff
@@ -273,3 +284,85 @@ func (r *ClusterDeploymentReconciler) createImageRegistry(ctx context.Context, r
 	}
 	return nil
 }
+
+// setACICapabilities will set the install config override annotation in the AgentClusterInstall if there
+// are additional enabled capabilities that need to be defined.
+// For SNO (single node openshift) and non-baremetal platform MNO (multi-node openshift), this is set if
+// the OpenshiftAssistedControlPlane has additional capabilities set.
+// For MNO (multi-node openshift), this is set to the default list for baremetal platform,
+// but the OpenshiftAssistedControlPlane can specify additional capabilities to be appended to this list.
+func setACICapabilities(oacp *controlplanev1alpha2.OpenshiftAssistedControlPlane, aci *hiveext.AgentClusterInstall) error {
+	isBaremetalPlatform := aci.Spec.PlatformType == hiveext.BareMetalPlatformType
+	if funk.IsEmpty(oacp.Spec.Config.Capabilities) && !isBaremetalPlatform {
```

Comment:
I think we shouldn't add the `funk` dependency just to check for zero value.

if we want to make it more readable, we can do something
```
if !areCapabilitiesSet(oacp) {
   return nil
}
....
```

Reply:
Good idea. Thanks! Added it in, what do you think?